Source/DafnyCore/Dafny.atg

```diff
@@ -188,6 +188,7 @@ public static Parser SetupParser(string/*!*/ s, Uri/*!*/ uri, ModuleDecl module,
   return new Parser(scanner, errors, module, builtIns);
 }
 
+// Unused code
```

Comment:
Any reason to keep this unused code at all?

Reply:
Well, It could well be handy, for example if an IDE allowed user's to write debugging expressions. So I did not want to just delete it because it is currently not called anywhere.